src/suites/cts/capability_info.ts

```diff
@@ -80,8 +80,8 @@ export const kPerStageBindingLimits: {
   'storage-texture': 4,
 };
 
-const kStagesAll = C.ShaderStage.Vertex | C.ShaderStage.Fragment | C.ShaderStage.Compute;
-const kStagesNonVertex = C.ShaderStage.Fragment | C.ShaderStage.Compute;
+const kStagesVFC = C.ShaderStage.Vertex | C.ShaderStage.Fragment | C.ShaderStage.Compute;
```

Comment:
nit: Would prefer kStagesAll and kStageCompute. The letters get hard to parse and overloaded since we import the constants as `C`